### PR TITLE
Added condition to make datatype a url

### PR DESCRIPTION
**Pull request by Vismayak**
I have only changed one file and though it was a simple change, I reformatted the file and moved the complex logic to another function in the same file

### CODE REVIEW

File: src/components/children/NestedInfoTable.jsx [hunk 1]
```diff
@@ -167,7 +167,7 @@ class NestedInfoTable extends React.Component {
 								})
 							) : key ==='dataType' ? // Another ternary operator to check if the value is a dataType if so create linking
 									(<TableCell>
-										<a href={`${config.hostname}/semantics/api/types/${data[key]}`} target="_blank" rel="noopener noreferrer">{data[key]} </a>
+										<a href={`/semantics/api/types/${data[key]}`} target="_blank" rel="noopener noreferrer">{data[key]} </a>
```
Comment (longshuicy) [inline src/components/children/NestedInfoTable.jsx:170]: 1. can you put the `/semantics/api/types/` part in config?
2. Could you also restrict this to only incore: or ergo: ones? maybe use some regex?

File: src/components/children/NestedInfoTable.jsx [hunk 2]
```diff
@@ -24,150 +24,161 @@ class NestedInfoTable extends React.Component {
...
-									{(typeof this.props.data[key] === "object" && this.props.data[key]) ?
-										Object.keys(this.props.data[key]).map((key2) => {
+		const startsWithErgoOrIncore = (datatype) => {
+			const regexErgo = /^ergo:/;
+			const regexIncore = /^incore:/;
+			return regexErgo.test(datatype) || regexIncore.test(datatype);
+		};
+		const renderDataRows = (data, classes, onClick) => {
+			const keysToExclude = ["hazardDatasets", "rasterDataset", "datasetId"];
+			console.log(`${config.hostname}semantics/api/types/` + "ergo");
```
Comment (longshuicy) [inline src/components/children/NestedInfoTable.jsx:39]: could you remove the console.log

File: src/app.config.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ let config = {};
 let hostname = process.env.INCORE_REMOTE_HOSTNAME || "";
 
 config["hostname"] = hostname;
-config["semanticService"] = "";
+config["semanticServiceType"] = `${hostname}/semantics/api/types`;
```
Comment (longshuicy) [inline src/app.config.js:5]: missing an ending "/" 
Or alternatively you can add that / when you do `${config.semanticServiceType}${data[key]}`

File: src/components/children/NestedInfoTable.jsx [hunk 4]
```diff
@@ -24,150 +24,161 @@ class NestedInfoTable extends React.Component {
...
+											)}
+										</TableRow>
+									);
+								})
+							) : key === "dataType" && startsWithErgoOrIncore(data[key]) ? ( // Another ternary operator to check if the value is a dataType if so create linking
+								<TableCell>
+									<a
+										href={`${config.semanticServiceType}${data[key]}`}
+										target="_blank"
+										rel="noopener noreferrer"
```
Comment (longshuicy) [inline src/components/children/NestedInfoTable.jsx:161]: Maybe try the theme color #e8a114
image
What do you think? up to you.

**Review by longshuicy — CHANGES_REQUESTED** [review]

**Review by longshuicy — APPROVED** [review]

**Review by ylyangtw — APPROVED** [review]
looks good

**Pull request merged**